src/version.js

```diff
@@ -19,7 +19,7 @@ function versionEqual(a, b) {
 function versionGreater(a, b) {
     return (!versionEqual(a, b))
         && zip(versionArray(a), versionArray(b)).reduce(
-            (prev, [a, b], index) => prev && (a >= (b || 0))
+            (prev, [a, b], index) => prev || (a > (b || 0))
```

Comment:
This would change `versionGreater("1.2", "2.1")` from `false` to `true`

We want all components of the version to be greater, not just one component.